hyperspy/signal.py

```diff
@@ -3301,21 +3313,24 @@ def fft(self, shifted=False, **kwargs):
         im_fft.change_dtype("complex")
         im_fft.metadata.General.title = 'FFT of {}'.format(
             im_fft.metadata.General.title)
-        im_fft.metadata.set_item('Signal.FFT.shifted', shifted)
+        im_fft.metadata.set_item('Signal.FFT.shifted', shift)
+        if hasattr(self.metadata.Signal, 'quantity'):
+            self.metadata.Signal.__delattr__('quantity')
```

Comment:
Why removing the quantity? Couldn't we compute the right one with pint? Same for units.

Reply:
I am not sure if pint can do it properly.